Doc/library/os.rst

```diff
@@ -193,6 +193,10 @@ process and user.
    to the environment made after this time are not reflected in :data:`os.environ`,
    except for changes made by modifying :data:`os.environ` directly.
 
+   The :meth:`!os.environ.refresh()` method updates
+   :data:`os.environ` with changes to the environment made in the same process
+   outside Python.
```

Comment:
```suggestion
   outside Python or by :func:`os.putenv`.
```

Reply:
Alternatively, `os.putenv()` could be wrapped to update `os.environ`.

Reply:
I applied your suggestion.